datalogistik/util.py

```diff
@@ -184,8 +196,9 @@ def convert_arrow_alias(type_name):
     return aliases.get(type_name, type_name)
 
 
-# Create an instance of the pyarrow datatype with the given name
 def arrow_type_function_lookup(function_name):
```

Comment:
```suggestion
def arrow_type_function_lookup(function_name: str) -> pa.Type:
```
or similar probably?

Reply:
I wasn't sure - what is the type of the result of `getattr`?